<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Dell-PC" name="practice.tc1sikuli" tests="1" failures="0" timestamp="19 Sep 2019 16:38:51 GMT" time="11.524" errors="1">
  <testcase name="data1" time="11.524" classname="practice.tc1sikuli">
    <error type="org.sikuli.script.FindFailed" message="C:/Users/Dell/Desktop/ss.PNG: (93x43) in S(0)[0,0 1366x768] E:Y, T:3.0">
      <![CDATA[FindFailed: C:/Users/Dell/Desktop/ss.PNG: (93x43) in S(0)[0,0 1366x768] E:Y, T:3.0
  Line 2761, in file Region.java

	at org.sikuli.script.Region.wait(Region.java:2761)
	at org.sikuli.script.Region.find(Region.java:2326)
	at org.sikuli.script.Region.getLocationFromTarget(Region.java:3220)
	at org.sikuli.script.Region.click(Region.java:3718)
	at org.sikuli.script.Region.click(Region.java:3703)
	at practice.tc1sikuli.data1(tc1sikuli.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- data1 -->
</testsuite> <!-- practice.tc1sikuli -->
